pkg/rancher-desktop/vue.config.js

```diff
@@ -36,7 +36,8 @@ module.exports = {
       .loader('raw-loader');
 
     config.plugin('define-plugin').use(webpack.DefinePlugin, [{
-      'process.client': JSON.stringify(true),
+      'process.client':       JSON.stringify(true),
+      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV || 'development'),
```

Comment:
This logic doesn't match the top of the file:

https://github.com/rancher-sandbox/rancher-desktop/blob/931d2701923956963ad0b0e59e6aade48d5093e2/pkg/rancher-desktop/vue.config.js#L10

It might be a good idea to make them match (one way or the other) to reduce confusion, or otherwise have a comment about it?

Reply:
We shouldn't need the mode specification at all, as it should default to what we expect via vue-cli (defaults to `development`/`production` with `serve/build` respectively)

https://cli.vuejs.org/guide/cli-service.html#vue-cli-service-serve